### PR TITLE
Copy of trial project

### PR DESCRIPTION
**Pull request by SelmaNdapanda**
In this pull request:

- I created the portfolio to showcase my skills

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    position: fixed;
+    right: .1%;
+    background-color:rgba(65, 65, 65, 0.8);
+    border-radius: 1px;
+    color: rgb(228, 228, 236);
+    font-size: 1.5rem;
+    opacity: .6;
+    top: 0;
+    padding: 0px 5px 0px 5px;
+    }  */
```
Comment (SelmaNdapanda) [inline style.css:44]: Please Do not leave unused code as comments in your codebase. Unused code as comments can make your code look unclean.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<html lang="en">
+  
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <link rel="stylesheet" href="style.css">
+      <title>Project</title>
```
Comment (SelmaNdapanda) [inline index.html:7]: Please give meaningful names to your Projects. Meaningful names can clearly distinguish one project from another.

**Review by SelmaNdapanda — COMMENTED** [review]
# Changes Required ♻

- [ ]  Use correct GitHub flow
- [ ]  Use descriptive README File
- [ ] Set up Linters

Please check all my comments below and make required changes.

Submit for a re-review when you are done.
Thanks 😊

File: style.css (hunk 1 above)
Comment (codecaiine) [reply style.css:44]: Your comments are polite and you use why :+1:

File: index.html (hunk 2 above)
Comment (codecaiine) [reply index.html:7]: Your comments are polite and you use why +1

**Comment by codecaiine** [comment]
Hi @SelmaNdapanda , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Great review :heavy_check_mark: 
- Good use of inline comments :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] **Please note that you could have marked the PR as invalid for the reason that Linter result is not green, and didn't pass checks in the PR.**
- [ ] Please use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**